ChangeLog

```diff
@@ -14,6 +24,7 @@
 	* man/BermudanSwaption.Rmd: Modified to reflect DiscountCurve
 	option and example updated
 	
+>>>>>>> eddelbuettel/master
```

Comment:
If you MUST reply by email, please try to delete all the quoted crap.  

Else it makes it all the way [here](https://github.com/eddelbuettel/rquantlib/pull/47#discussion_r63219879) and clutters the thread.

And for the record, no screenshot attached.  I don;t really care either way.  I will just ignore malformed PRs.